vmms/ec2SSH.py

```diff
@@ -244,7 +299,7 @@ def waitVM(self, vm, max_secs):
             # If the call to ssh returns timeout (-1) or ssh error
             # (255), then success. Otherwise, keep trying until we run
             # out of time.
-            ret = timeout(["ssh"] + Ec2SSH._SSH_FLAGS +
+            ret = timeout(["ssh"] + self.ssh_flags +
```

Comment:
Thanks for this change! It's better style to have a `ssh_flags` property rather than "hardcode" the `Ec2SSH_SSH_FLAGS` property everywhere